docusaurus/docs/guides/Identity_Providers/auth0/index.md

```diff
@@ -132,3 +257,8 @@ Take note of the `jwks_uri` value returned from the above openid-configuration e
 ### Create External JWT Signer
 Using the values described above, use the `ziti` CLI to configure an external JWT signer that represents your Auth0 identity provider.  You can find details on how to do this in the [BrowZer Quickstart documentation](/docs/learn/quickstarts/browzer/)
 
+`IMPORTANT NOTE:`
+
+For Ziti External JWT Signer's that refer to Auth0, the `audience` value must be the same value you used to create the above Auth0 `API`. For the example used in this documentation, it is `https://demo.ziti.netfoundry.io`.  
```

Comment:
would you remove "Ziti" here pls? also "External JWT Signers" all capitalized seems like External and Signers should be lowercased